source/iostreams.tex

```diff
@@ -6892,7 +6892,10 @@
 Then, if \tcode{os.rdbuf()} is a
 \tcode{basic_syncbuf<charT, traits, Allocator>*},
 called \tcode{buf} for the purpose of exposition,
-calls \tcode{buf->emit()}.
+behaves as an unformatted output function\iref{ostream.unformatted}
+of \tcode{os}.
+After constructing a \tcode{sentry} object, calls \tcode{buf->emit()}.
```

Comment:
...but we should probably be consistent in this PR?

Reply:
Yeah, maybe. I've created #5031 for the general fix, but given the direction there, I'd like to keep this as-is.